keps/sig-node/4818-allow-zero-value-for-sleep-action-of-prestop-hook/README.md

```diff
@@ -342,6 +224,12 @@ Basic functionality
 - Delete the pod and observe the time it takes for the container to terminate.
 - Verify that the container terminates immediately without sleeping.
 
+Additional e2e tests for beta:
```

Comment:
Do any tests exist for this feature for alpha? And can we point to any presubmits or an existing job?